File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.js
function solution(s) {
    const num = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine'];
    num.forEach((v, i) => {
        s = s.replaceAll(v, i);
    })
    return parseInt(s);
}

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    num = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
    for i, v in enumerate(num):
        s = s.replace(v, str(i))
    return int(s)